app/src/main/java/org/wikipedia/analytics/metricsplatform/MetricsPlatform.kt

```diff
@@ -3,22 +3,37 @@ package org.wikipedia.analytics.metricsplatform
 import org.wikimedia.metrics_platform.MetricsClient
 import org.wikimedia.metrics_platform.context.AgentData
 import org.wikimedia.metrics_platform.context.ClientData
+import org.wikimedia.metrics_platform.context.MediawikiData
 import org.wikipedia.WikipediaApp
 import org.wikipedia.settings.Prefs
+import org.wikipedia.util.ReleaseUtil
 import java.time.Duration
 
 object MetricsPlatform {
-    private val agentData = AgentData(
+    val agentData = AgentData(
         WikipediaApp.instance.appInstallID,
         "android",
         "app"
     )
 
+    val mediawikiData = MediawikiData(
+        WikipediaApp.instance.currentTheme.toString(),
+        WikipediaApp.instance.versionCode.toString(),
+        ReleaseUtil.isProdRelease,
+        ReleaseUtil.isDevRelease,
+        WikipediaApp.instance.wikiSite.dbName(),
+        WikipediaApp.instance.languageState.systemLanguageCode,
+        null
```

Comment:
I err'd on the side of sending this data as part of the MW data object -- this null represents "siteContentLanguageVariant" << does it make sense to put anything here or is null appropriate?